src/evergreen/resource_permissions.py

```diff
@@ -0,0 +1,47 @@
+# -*- encoding: utf-8 -*-
```

Comment:
Could you add some tests for this code?

For models like this what we have been doing is:
1. Adding some sample json to the [tests/evergreen/data](https://github.com/evergreen-ci/evergreen.py/tree/main/tests/evergreen/data) folder.
2. Add a pytest fixture to read the sample data (e.g. [here](https://github.com/evergreen-ci/evergreen.py/blob/babf467ab7a271977b28680ccb350172c259c069/tests/conftest.py#L40-L43).
3. Read a few of the attributes and any interesting states that might be returned (e.g. [here](https://github.com/evergreen-ci/evergreen.py/blob/babf467ab7a271977b28680ccb350172c259c069/tests/evergreen/test_commitqueue.py#L9-L12).